src/sentry/models/organizationmemberteamreplica.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import (
+    BaseModel,
+    BoundedAutoField,
+    BoundedBigIntegerField,
+    control_silo_only_model,
+    sane_repr,
+)
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+
+
+@control_silo_only_model
+class OrganizationMemberTeamReplica(BaseModel):
```

Comment:
I know I'm late to the party @corps , but just curious: why not name this `OrganizationMemberTeamMapping`? I'd have expected the following:

| Region silo source model | Control silo "shadow" model |
| --- | --- |
| Organization | OrganizationMapping |
| OrganizationMember | OrganizationMemberMapping |
| OrganizationMemberTeam | OrganizationMemberTeamMapping |

Reply:
I love that you are asking these questions, and I will follow up tomorrow for sure!